activerecord/test/cases/tasks/database_tasks_test.rb

```diff
@@ -280,7 +280,10 @@ def test_migrate_receives_correct_env_vars
       ENV['VERBOSE'] = 'false'
       ENV['VERSION'] = '4'
 
-      ActiveRecord::Migrator.expects(:migrate).with(ActiveRecord::Migrator.migrations_paths, 4)
+      ActiveRecord::Tasks::DatabaseTasks.expects(:migrations_paths).
```

Comment:
Instead of expecting on the instance under test itself, can't we assign the value?

```ruby
original_migrations_path = ActiveRecord::Tasks::DatabaseTasks.migrations_paths
begin
  ActiveRecord::Tasks::DatabaseTasks.migrations_paths = "custom/path"
  ... 
ensure
  ActiveRecord::Tasks::DatabaseTasks.migrations_paths = original_migrations_path
end
```